web-client/README.md

```diff
@@ -2,7 +2,7 @@
 
 IronRDP also supports the web browser as a first class target.
 
-See the [iron-remote-gui](./iron-remote-gui) for the reusable Web Component, and [iron-svelte-client](./iron-svelte-client) for a demonstration.
+See the [iron-remote-desktop-rdp](./iron-remote-desktop-rdp) for the reusable Web Component, and [iron-svelte-client](./iron-svelte-client) for a demonstration.
```

Comment:
issue: The reusable web component is `iron-remote-desktop`.
```suggestion
See the [iron-remote-desktop](./iron-remote-desktop) for the reusable Web Component, and [iron-svelte-client](./iron-svelte-client) for a demonstration.
```
`iron-remote-desktop-rdp` would be the RDP backend based on IronRDP!

Reply:
Done